### PR TITLE
Added RA, Dec to Region Searching NB

### PR DESCRIPTION
**Pull request by ColinOrionChandler**
Added (RA, Dec) searching for (1) overlapping sets and (2) entire dataframe. Also cleaned up imports and a few other tidbits.

### CODE REVIEW

File: notebooks/region_search/Region Searching Workbook.ipynb [hunk 1]
```diff
@@ -2813,36 +2817,608 @@
...
+    "\n",
+    "    all_centers = []\n",
+    "    all_labels = []\n",
+    "\n",
+    "    id_to_coord = df.set_index(\"data_id\")[\"center_coord\"].to_dict()\n",
+    "\n",
+    "    for i, set_name in enumerate(list(overlapping_sets.keys())):\n",
+    "        all_labels.append(set_name)\n",
+    "        center_data_id = overlapping_sets[set_name][0]\n",
+    "        # print(f\"See set_name is {set_name} and center_data_id is {center_data_id}.\", flush=True)\n",
```
Comment (wilsonbb) [inline notebooks/region_search/Region Searching Workbook.ipynb:2878]: Can we either remove this line or have

```
if verbose:
    print(f"...")
```

File: notebooks/region_search/Region Searching Workbook.ipynb [hunk 2]
```diff
@@ -2813,36 +2817,608 @@
...
+    "        all_centers.append(center_coord)\n",
+    "\n",
+    "    all_centers = SkyCoord(\n",
+    "        ra=[x[0] for x in all_centers] * u.degree,\n",
+    "        dec=[x[1] for x in all_centers] * u.degree,\n",
+    "    )\n",
+    "\n",
+    "    distances = ra_dec_coord.separation(\n",
+    "        all_centers\n",
+    "    )  # .to(u.arcsec).value  # Convert distances to arcseconds as numeric values\n",
```
Comment (wilsonbb) [inline notebooks/region_search/Region Searching Workbook.ipynb:2889]: Does the comment  "# Convert distances to arcseconds as numeric values" still make sense. Seems we're no longer interested in converting to arcseconds here

**Review by wilsonbb — APPROVED** [review]
Some cool plots! Some small comments

File: notebooks/region_search/Region Searching Workbook.ipynb (hunk 2 above)
Comment (ColinOrionChandler) [reply notebooks/region_search/Region Searching Workbook.ipynb:2889]: Removed, thanks!

File: notebooks/region_search/Region Searching Workbook.ipynb (hunk 1 above)
Comment (ColinOrionChandler) [reply notebooks/region_search/Region Searching Workbook.ipynb:2878]: Removed, this was for debugging. Too much spamming if we leave it in for verbose even.

**Pull request merged**